cdap-source-control/src/main/java/io/cdap/cdap/sourcecontrol/ApplicationManager.java

```diff
@@ -0,0 +1,84 @@
+/*
```

Comment:
Moved the interface to source-control module as discussed. This allows the interface to be used by `InMemorySourceControlRunner`. The implementations and tests will remain in app-fabric